prov/terraform/commons/utils.go

```diff
@@ -12,16 +12,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package google
+package commons
 
 import (
 	"crypto/sha1"
 	"fmt"
-
 	"github.com/ystia/yorc/v4/config"
 )
 
-func getResourcesPrefix(cfg config.Configuration, deploymentID string) string {
+// GetResourcesPrefix : TODO
+func GetResourcesPrefix(cfg config.Configuration, deploymentID string) string {
```

Comment:
For me this function is specific to google.
We added this because Google Cloud does not allow to have resources with the same name.
This limitation doesn't exist on OpenStack & AWS.

Reply:
Oh ok ! I wasn't sure, so I've done it just in case. I'll rollback.